tests/integration/blockchain/test_blockchain_reorganization.py

```diff
@@ -13,21 +17,65 @@ class BlockchainReorganizationTests(IntegrationTestCase):
         )
 
     async def test_reorg(self):
+        REORG_COUNT.set(0)
         # invalidate current block, move forward 2
-        self.assertEqual(self.ledger.headers.height, 200)
-        await self.assertBlockHash(200)
-        await self.blockchain.invalidate_block((await self.ledger.headers.hash(200)).decode())
+        self.assertEqual(self.ledger.headers.height, 206)
+        await self.assertBlockHash(206)
+        await self.blockchain.invalidate_block((await self.ledger.headers.hash(206)).decode())
         await self.blockchain.generate(2)
-        await self.ledger.on_header.where(lambda e: e.height == 201)
-        self.assertEqual(self.ledger.headers.height, 201)
-        await self.assertBlockHash(200)
-        await self.assertBlockHash(201)
+        await self.ledger.on_header.where(lambda e: e.height == 207)
+        self.assertEqual(self.ledger.headers.height, 207)
+        await self.assertBlockHash(206)
+        await self.assertBlockHash(207)
+        self.assertEqual(1, REORG_COUNT._samples()[0][2])
 
         # invalidate current block, move forward 3
-        await self.blockchain.invalidate_block((await self.ledger.headers.hash(200)).decode())
+        await self.blockchain.invalidate_block((await self.ledger.headers.hash(206)).decode())
         await self.blockchain.generate(3)
-        await self.ledger.on_header.where(lambda e: e.height == 202)
-        self.assertEqual(self.ledger.headers.height, 202)
-        await self.assertBlockHash(200)
-        await self.assertBlockHash(201)
-        await self.assertBlockHash(202)
+        await self.ledger.on_header.where(lambda e: e.height == 208)
+        self.assertEqual(self.ledger.headers.height, 208)
+        await self.assertBlockHash(206)
+        await self.assertBlockHash(207)
+        await self.assertBlockHash(208)
+        self.assertEqual(2, REORG_COUNT._samples()[0][2])
+
+    async def test_reorg_dropping_claim(self):
+        # sanity check
+        result_txs, _, _, _ = await self.ledger.claim_search([], name='hovercraft')
+        self.assertListEqual(result_txs, [])
+
+        # create a claim and verify it is returned by claim_search
+        self.assertEqual(self.ledger.headers.height, 206)
+        broadcast_tx = Transaction(unhexlify((await self.stream_create(name='hovercraft'))['hex'].encode()))
+        self.assertEqual(self.ledger.headers.height, 207)
+        result_txs, _, _, _ = await self.ledger.claim_search([], name='hovercraft')
+        self.assertEqual(1, len(result_txs))
+        tx = result_txs[0]
+        self.assertEqual(tx.tx_ref.id, broadcast_tx.id)
```

Comment:
variable name would be clearer as `txo` (that is what claim_search returns) then `txo.tx_ref.id` makes more sense